### ISSUE TITLE
Title and role attributes are missing for functional control on Home screen

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#796]
**Describe the bug**
Title and role attributes are missing for functional control on Home screen. In Home View, there are different tabs such as "Add New", "Search", "Browse Library". When we move mouse over it, there is not label/title displayed. 

**To Reproduce**
1. Go to Home screen (http://localhost:9992/cm/app/?view=home&pathType=unknown)
2. Hover over different tabs such as "Add New", "Search", "Browse Library".
3. No title/label displayed.

**Expected behavior**
Title/Label should be displayed on each component of the screen.
